lib/cn/cn_tree.c

```diff
@@ -3253,6 +3272,8 @@ cn_comp(struct cn_compaction_work *w)
 
 /**
  * cn_comp_cancel_cb() - sts callback when a job is canceled
+ *
+ * [HSE_REVISIT] Only used by unit tests, should we remove it?
```

Comment:
YES! Alternatively move it to test-specific code.

Reply:
Just didn't know if we wanted to retain the "cancel" functionality for the future use.  But probably should just remove it...  Alex?

Reply:
Right, but this function isn't called from anywhere, but cw_canceled is set either from a failed job or cn_get_cancel(), so I'm not removing that, just this function.  Curiously, in cn_comp_compact() why do we run the job before wating to the end?